ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/ovn/fips/efs/cucushift-installer-rehearse-aws-ipi-ovn-fips-efs-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: cucushift-installer-rehearse-aws-ipi-ovn-fips-efs
```

Comment:
The workflow removed "FIPS_ENABLED" ENV, but we are expecting the workflow cover fips.

Reply:
Good catch. Sorry, I missed it. Updated.